src/SearchResults.js

```diff
@@ -14,14 +15,29 @@ export default function Search({
   fetchSearchResults,
   showHeader = false,
 }) {
+  // eslint-disable-next-line camelcase
+  const page_size = 5;
```

Comment:
no need to disable linter here. convention in JS is to always use camelcase.

Reply:
I was anticipating feedback on this!

The reason I went with `page_size` and went through the trouble of commenting `// eslint-disable-next-line camelcase` in a couple of different places is so that I could do
```
  ] = useState({
    ...SOUND_LIST_QUERY_PARAMS,
    page_size,
  });
```
instead of
```
  ] = useState({
    ...SOUND_LIST_QUERY_PARAMS,
    page_size: limit,
  });
```
since `page_size` is the actual name [the Freesound API](https://freesound.org/docs/api/resources_apiv2.html#response-sound-list) uses as one of its request parameters.

I'm not really committed to either approach, so I'll defer to your judgment on whether we should stay with `page_size` to make the request parameters more succinct or do `page_size: limit` to follow JavaScript conventions.

Reply:
JS client APIs that make network requests are responsible for normalizing casing so that consumers of that client library can use idiomatic casing conventions in their code. The goal of the client library is that consumers of it should never have to read the freesound api docs.

Reply:
So might be best to allow casing conversion in freesound-client, by automatically converting params from camelcase to snakecase automatically. We shouldn't need to worry about that complexity here.